### ISSUE TITLE
Exception when -f file does not exist

### ISSUE THREAD

**Issue by kamsz** [roboll/helmfile#48]
```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
        panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x56a170]

goroutine 1 [running]:
github.com/roboll/helmfile/vendor/github.com/urfave/cli.HandleAction.func1(0xc4200476f8)
        /Users/kamsz/go/src/github.com/roboll/helmfile/vendor/github.com/urfave/cli/app.go:474 +0x2c8
panic(0x59aa40, 0x6b78e0)
        /usr/local/Cellar/go/1.9.3/libexec/src/runtime/panic.go:491 +0x283
main.before(0xc4200ae280, 0xc42000a800, 0x20, 0x18, 0xc42008e5d0, 0xc42008e5d0)
        /Users/kamsz/go/src/github.com/roboll/helmfile/main.go:251 +0x150
main.main.func3(0xc4200ae280, 0x0, 0x0)
        /Users/kamsz/go/src/github.com/roboll/helmfile/main.go:140 +0x40
reflect.Value.call(0x591ec0, 0x5d2e20, 0x13, 0x5c75b8, 0x4, 0xc420047698, 0x1, 0x1, 0xc420047618, 0x5bf120, ...)
        /usr/local/Cellar/go/1.9.3/libexec/src/reflect/value.go:434 +0x905
reflect.Value.Call(0x591ec0, 0x5d2e20, 0x13, 0xc420047698, 0x1, 0x1, 0xc4200ae280, 0x7f2112147ee0, 0x0)
        /usr/local/Cellar/go/1.9.3/libexec/src/reflect/value.go:302 +0xa4
github.com/roboll/helmfile/vendor/github.com/urfave/cli.HandleAction(0x591ec0, 0x5d2e20, 0xc4200ae280, 0x0, 0x0)
        /Users/kamsz/go/src/github.com/roboll/helmfile/vendor/github.com/urfave/cli/app.go:483 +0x180
github.com/roboll/helmfile/vendor/github.com/urfave/cli.Command.Run(0x5c7640, 0x4, 0x0, 0x0, 0x0, 0x0, 0x0, 0x5d162b, 0x33, 0x0, ...)
        /Users/kamsz/go/src/github.com/roboll/helmfile/vendor/github.com/urfave/cli/command.go:186 +0x9ed
github.com/roboll/helmfile/vendor/github.com/urfave/cli.(*App).Run(0xc420001080, 0xc42000e080, 0x4, 0x4, 0x0, 0x0)
        /Users/kamsz/go/src/github.com/roboll/helmfile/vendor/github.com/urfave/cli/app.go:237 +0x57d
main.main()
        /Users/kamsz/go/src/github.com/roboll/helmfile/main.go:229 +0xbd5
```

**Comment by mumoshu** [comment]
@kamsz Thanks for the report 👍 
Seems like I've missed an `else` block here: https://github.com/roboll/helmfile/blob/e4961a8c34c5c0f13afe7d7127904a7695ddaf59/main.go#L250

There should be something like:

```golang
} else {
    return nil, nil, fmt.Errorf("failed to read %s: %v", file, err)
}
```

Would you mind contributing a fix?

**Comment by mumoshu** [comment]
I believe this is resolved via #83 thanks to @sstarcher
